O3exp/pipeline/data_O3_C00.ini

```diff
@@ -0,0 +1,38 @@
+[workflow]
+; http://ligo-cbc.github.io/pycbc/releases/v1.2.0/html/workflow/initialization.html
+h1-channel-name = H1:GDS-CALIB_STRAIN
+l1-channel-name = L1:GDS-CALIB_STRAIN
+v1-channel-name = V1:Hrec_hoft_16384Hz
```

Comment:
Not clear why V1 is not commented out here as it is in ifos. Add a comment explaining that the V1 options here are ignored unless turned on in ifos.

In general prefer not to have options that are ignored, but convenience may override that here